Cargo.toml

```diff
@@ -58,6 +72,7 @@ sha3 = { version = "0.10", default-features = false }
 winter-crypto = { version = "0.10", default-features = false }
 winter-math = { version = "0.10", default-features = false }
 winter-utils = { version = "0.10", default-features = false }
+rayon = { version = "1.10.0", optional = true }
```

Comment:
nit: we usually don't specify the patch version (i.e., should be `1.10` rather than `1.10.0`) and also alphabetize dependencies.